awt-graphics-rest-quickstart/src/main/docker/Dockerfile.native-distroless

```diff
@@ -1,24 +1,4 @@
 ####
-# ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
-# ████████████░▄▄▀█░▄▄█░███░█░▄▄▀█░▄▄▀█░▄▄█████████████
-# ████████████░▄▄▀█░▄▄█▄▀░▀▄█░▀▀░█░▀▀▄█░▄▄█████████████
-# ████████████░▀▀░█▄▄▄██▄█▄██▄██▄█▄█▄▄█▄▄▄█████████████
-# ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
-# Preparing distroless Dockerfile in this fashion is highly
-# application specific.
-#
-# Using Debian based https://github.com/quarkusio/quarkus-images/blob/main/quarkus-distroless-image.yaml#L16
-# will not do for this Quickstart as only libz.so is copied and the runtime
-# might not be compatible with Quarkus executable built using UBI image.
-#
-# The undermentioned example uses ubi-micro image and manually copies necessary
-# libraries and files from ubi-minimal image.
-#
-# See https://www.redhat.com/en/blog/introduction-ubi-micro on how to start using Buildah.
-#
```

Comment:
@matthyx May I ask you to leave the info there, just deleting 
```
# Using Debian based https://github.com/quarkusio/quarkus-images/blob/main/quarkus-distroless-image.yaml#L16
# will not do for this Quickstart as only libz.so is copied and the runtime
# might not be compatible with Quarkus executable built using UBI image.
```

The pointer about buildah is still relevant. Copying files manually in this fashion does the job for a simple example, for the sake of brevity, but it won't do for any serious long term maintainable project.